src/sema/function_annotation.rs

```diff
@@ -38,6 +151,7 @@ pub fn function_annotations(
 
     for note in annotations {
         match note.id.name.as_str() {
+            "selector" => (), // selectors done in function_prototype_annotations
```

Comment:
Why?

Reply:
We have `fn function_prototype_annotations()` for annotations we can resolve without a symbol table (i.e. parameters). `fn function_body_annotations()` are with a symbol table. I've added some comments to the functions. Do you think this is clear?